### PR TITLE
Return raw library

### PR DESCRIPTION
**Pull request by tjacovich**
Gives users the option to return raw bibcodes from a library instead of the bibcodes returned from `bigquery`

### CODE REVIEW

File: biblib/views/http_errors.py [hunk 1]
```diff
@@ -53,7 +53,7 @@
 )
 SOLR_RESPONSE_MISMATCH_ERROR = dict(
     body='Solr response does not contain the same number of bibcodes as the '
-         'request.',
+         'request, or only the raw library was requested.',
```
Comment (kelockhart) [inline biblib/views/http_errors.py:56]: Is this text update needed? As far as I can tell, this error is only used within a `if not raw_library` block

File: biblib/views/library_view.py [hunk 2]
```diff
@@ -327,35 +333,45 @@ def get(self, library):
...
+                    documents.sort()
+                    documents = documents[start:start+rows]
+            
             else:
-                # Some problem occurred, we will just ignore it, but will
-                # definitely log it.
-                solr = SOLR_RESPONSE_MISMATCH_ERROR['body']
-                current_app.logger.warning('Problem with solr response: {0}'
-                                           .format(solr))
+                solr = 'Only the raw library was requested.'
```
Comment (kelockhart) [inline biblib/views/library_view.py:372]: Add a unittest for this `else` block

**Review by kelockhart — CHANGES_REQUESTED** [review]
Let's add a unittest for the `raw_library` option, but otherwise looks good.

**Review by kelockhart — APPROVED** [review]
Looks good!

**Comment by tjacovich** [comment]
Awesome. I'll merge and release today and deploy to `dev` Monday.

**Pull request merged**